kensu/sdk/__init__.py

```diff
@@ -361,3 +375,25 @@ def get_datasources(self):
 
     def get_logical_datasources(self):
         return self.requests_get_json("/business/services/views/v1/logical-datasources")
+
+    def get_total_sent_data_mb(self):
+        return self.requests_get_json("/business/services/v1/ingestion-log")['entitiesSentKbTotal'] / 1000
+
+    def get_detailed_sent_data(self):
+        ingestion_log_details = self.requests_get_json("/business/services/v1/ingestion-log")['entitiesSentByToken']
```

Comment:
do we tag these endpoints somewhere as used externally (so our backenders would be well aware)?

Reply:
good question Vidma...
@leszekgruchala do we have a procedure for this?

Reply:
@emaluckensu You can let us know on Slack. But I also see it already defined here https://kensu.atlassian.net/browse/DAM-7830